scripts/mutate.py

```diff
@@ -131,9 +131,13 @@ def mutate_integer_literal(value_src: str) -> str:
         if value == 0:
             mutated_value = 1
         else:
-            mutated_value = random.choice([0, 1, value + 1, value - 1])
+            options_list = [0, 1, value + 1, value - 1]
```

Comment:
Doh, I was in the process of submitting a PR for this but you beat me to it :+1:. I can still submit a brief one including the logic for skipping assertions though.

Reply:
Ah yeah, no worries! I applied a quick patch so that I can get rid of the numerous false alarms while working on this PR, but I'll revert it before merging. Feel free to open a separate PR :smile: